clang/include/clang/StaticAnalyzer/Core/PathSensitive/SValBuilder.h

```diff
@@ -209,6 +209,12 @@ class SValBuilder {
                                         const LocationContext *LCtx,
                                         QualType type,
                                         unsigned visitCount);
+  DefinedOrUnknownSVal conjureSymbolVal(const CallEvent &call, QualType type,
+                                        unsigned visitCount,
+                                        const void *symbolTag = nullptr);
```

Comment:
Shouldn't the Call already have a type? Why do we need to pass it separately?

Reply:
Some callsites use a type that is different from the result type of the call, so I'm trying to preserve the behaviour of the code.